src/js/utils/dateUtils.js

```diff
@@ -0,0 +1,16 @@
+import moment from 'moment';
+
+const dateWithoutTimeZone = ({
```

Comment:
I'm fine with this for now since likely the discovery will revisit this util file, but this isn't really a date without a timezone, it's a date in utc with the same time. So maybe a name similar to "dateInUtcPreserveTime" makes more sense